### ISSUE TITLE
softmax in cifar10

### ISSUE THREAD

**Issue by AlanZhangv1** [kevinlin311tw/Caffe-DeepBinaryCode#15]
hi Kevin,I have a problem about the softmax loss function in CIFAR10 dataset.It seems that you don't use softmax to predict the probility for every class ?and direct use the fc9 outputs to calculate the accuracy?

**Comment by kevinlin311tw** [comment]
We use softmax to predict the probability. In the testing (or validation) phase, we use the accuracy layer in caffe which help us compute the probability as well as evaluate the classification accuracy.
